linter.py

```diff
@@ -62,10 +62,8 @@ def find_errors(self, output):
             for warning in data['warnings']:
                 line = warning['line'] - self.line_col_base[0]
                 col = warning['column'] - self.line_col_base[1]
-                type = warning['severity']
-                text = warning['text']
+                severity = warning['severity']
+                rule = warning['rule']
+                text = warning['text'].replace('(' + rule + ')', '')
```

Comment:
unsure, but maybe `.rstrip()` for trailing whitespace

Reply:
Good idea, I guess it leaves a trailing space now.